src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -653,6 +653,28 @@ def is_mapper_device(device_name):
     return device_name.startswith(('/dev/mapper', '/dev/dm-'))
 
 
+def is_writable_raw_device(disk_path):
+    '''
+    Can we open the device in exclusive mode ?
```

Comment:
Try not to use personal pronouns when writing docs (who is "we" ? who is "someone" ?) An impersonal docstring could be: 

`Check if device can be opened in exclusive mode`

Reply:
fixed too